### PR TITLE
SITES-657: Uninstall webauth_extras

### PR DESCRIPTION
**Pull request by jbickar**
# READY FOR REVIEW

# Summary
- Just what it says on the tin

# Needed By (Date)
- Meh. End of August?

# Criticality
- How critical is this PR on a 1-10 scale? 1/10

# Steps to Test

1. Choose a site to migrate that has `webauth_extras` enabled
2. Check out this branch
3. Migrate the site
4. Ensure that the status of `webauth_extras` is "Not installed"

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules? ACSF migration

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/SITES-657

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: roles/upload-site/tasks/main.yml [hunk 1]
```diff
@@ -120,6 +120,7 @@
     # Must truncate this table otherwise users will lose roles in WMD->SSP upgrade path.
     - "sqlq 'truncate table webauth_roles_history'"
     - "sspwmd"
+    - "-y pm-uninstall webauth_extras"
```
Comment (sherakama) [inline roles/upload-site/tasks/main.yml]: This module needs to be disabled before it can be uninstalled.

Comment (sherakama) [reply roles/upload-site/tasks/main.yml]: Why not roll this up in to lines 117 and 118?

File: roles/upload-site/tasks/main.yml [hunk 2]
```diff
@@ -120,6 +120,7 @@
     # Must truncate this table otherwise users will lose roles in WMD->SSP upgrade path.
     - "sqlq 'truncate table webauth_roles_history'"
     - "sspwmd"
```
Comment (jbickar) [inline roles/upload-site/tasks/main.yml]: It gets disabled here.

File: roles/upload-site/tasks/main.yml (hunk 1 above)
Comment (jbickar) [reply roles/upload-site/tasks/main.yml]: See comment on line 122.

Comment (jbickar) [reply roles/upload-site/tasks/main.yml]: `GOTO 122`

Comment (sherakama) [reply roles/upload-site/tasks/main.yml]: Oh, I see your note in Jira re: disabled due to webauth being disabled. How about a belt and suspenders approach and we add it to lines 117 and 118 anyway?

Comment (sherakama) [reply roles/upload-site/tasks/main.yml]: Gotcha, needs it for drush command.

Comment (jbickar) [reply roles/upload-site/tasks/main.yml]: Well, we don't technically need `webauth_extras` for the drush command, but whatevs, it's merged now.

**Pull request merged**